src/helpers/authStore.js

```diff
@@ -0,0 +1,35 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+Vue.use(Vuex)
+
+const state = {
+  user: null,
```

Comment:
Initialize the state with a proper representation of what that will be. User should be a object for example.

Check this here: http://vuex.vuejs.org/en/mutations.html#mutations-follow-vues-reactivity-rules